implementation/pom.xml

```diff
@@ -54,4 +54,21 @@
       <scope>provided</scope>
     </dependency>
   </dependencies>
+
+  <profiles>
+    <profile>
+      <id>coverage</id>
```

Comment:
Is this supposed to report coverage on PRs?

Reply:
Unfortunately, it's a by-product of CI in that the job is the same for master and a PR.

If you know a way to prevent that I'm all ears